### PR TITLE
Feature/storybook

### PR DESCRIPTION
**Pull request by jelleM**

### CODE REVIEW

File: storybook/stories/Button/index.android.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```
Comment (JasperVercammen) [inline storybook/stories/Button/index.android.js]: Is it needed to make this file for android and ios separately?

Comment (jelleM) [reply storybook/stories/Button/index.android.js]: Probably not, I will remove them

File: package.json [hunk 2]
```diff
@@ -1,10 +1,16 @@
 {
   "name": "reactnative_boilerplate",
-  "version": "0.1.0",
+  "version": "1.7.0",
```
Comment (vanlooverenkoen) [inline package.json]: Why update the boilerplace version?

File: storybook/index.android.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import StorybookUI from './storybook';
```
Comment (vanlooverenkoen) [inline storybook/index.android.js]: why is there an android and iOS and index file with the same code? Isn't it possible to remove android and ios files

File: package.json (hunk 2 above)
Comment (jelleM) [reply package.json]: Strange, probably a typo. Didn't mean to change it

File: storybook/index.android.js (hunk 3 above)
Comment (jelleM) [reply storybook/index.android.js]: Not needed indeed, I removed them.

**Pull request closed without merging**